go.mod

```diff
@@ -162,7 +162,7 @@ replace github.com/cosmos/iavl => github.com/cosmos/iavl v0.17.3
 replace github.com/ethereum/go-ethereum => github.com/crypto-org-chain/go-ethereum v1.10.3-patched
 
 // TODO: remove when ibc-go and ethermint upgrades cosmos-sdk
-replace github.com/cosmos/cosmos-sdk => github.com/cosmos/cosmos-sdk v0.44.7-0.20220214161517-8a26cd10b4be
+replace github.com/cosmos/cosmos-sdk => github.com/yihuang/cosmos-sdk v0.44.6-cronos-revert
```

Comment:
https://github.com/yihuang/cosmos-sdk/compare/8a26cd10b4be...v0.44.6-cronos-revert?expand=1

TODO: push to crypto-org-chain fork after verified.